src/main/java/org/jabref/migrations/PreferencesMigrations.java

```diff
@@ -297,20 +299,27 @@ private static void upgradeObsoleteLookAndFeels() {
         String currentLandF = prefs.get(JabRefPreferences.WIN_LOOK_AND_FEEL);
 
         Stream.of("com.jgoodies.looks.windows.WindowsLookAndFeel", "com.jgoodies.looks.plastic.PlasticLookAndFeel",
-                "com.jgoodies.looks.plastic.Plastic3DLookAndFeel", "com.jgoodies.looks.plastic.PlasticXPLookAndFeel",
-                "com.sun.java.swing.plaf.gtk.GTKLookAndFeel")
-                .filter(style -> style.equals(currentLandF))
-                .findAny()
-                .ifPresent(loolAndFeel -> {
-                    if (OS.WINDOWS) {
-                        String windowsLandF = "com.sun.java.swing.plaf.windows.WindowsLookAndFeel";
-                        prefs.put(JabRefPreferences.WIN_LOOK_AND_FEEL, windowsLandF);
-                        LOGGER.info("Switched from obsolete look and feel " + currentLandF + " to " + windowsLandF);
-                    } else {
-                        String nimbusLandF = "javax.swing.plaf.nimbus.NimbusLookAndFeel";
-                        prefs.put(JabRefPreferences.WIN_LOOK_AND_FEEL, nimbusLandF);
-                        LOGGER.info("Switched from obsolete look and feel " + currentLandF + " to " + nimbusLandF);
-                    }
-                });
+                  "com.jgoodies.looks.plastic.Plastic3DLookAndFeel", "com.jgoodies.looks.plastic.PlasticXPLookAndFeel",
+                  "com.sun.java.swing.plaf.gtk.GTKLookAndFeel")
+              .filter(style -> style.equals(currentLandF))
+              .findAny()
+              .ifPresent(loolAndFeel -> {
+                  if (OS.WINDOWS) {
+                      String windowsLandF = "com.sun.java.swing.plaf.windows.WindowsLookAndFeel";
+                      prefs.put(JabRefPreferences.WIN_LOOK_AND_FEEL, windowsLandF);
+                      LOGGER.info("Switched from obsolete look and feel " + currentLandF + " to " + windowsLandF);
+                  } else {
+                      String nimbusLandF = "javax.swing.plaf.nimbus.NimbusLookAndFeel";
+                      prefs.put(JabRefPreferences.WIN_LOOK_AND_FEEL, nimbusLandF);
+                      LOGGER.info("Switched from obsolete look and feel " + currentLandF + " to " + nimbusLandF);
+                  }
+              });
+    }
+
+    static void upgradePreviewStyleFromReviewToComment() {
+        JabRefPreferences prefs = Globals.prefs;
```

Comment:
You can pass the preferences class as a parameter and then use `mock` in combination with `verify` in the tests below (instead of working on the actual preference object).

Reply:
I tried around with this but I currently see no way to do this. When I mock the object, calling the migration method on the object will have no effect as it operates directly on the preferences object. I don't like that operating on the real prefs object but 
Sure I can mock returns, but that's not the goal of this test. 
However, I could remove the dependency on the actual Globals.prefs object by passing a Preferences object as parametre